frontend/app_flowy/lib/workspace/presentation/widgets/float_bubble/question_bubble.dart

```diff
@@ -39,6 +44,50 @@ class QuestionBubble extends StatelessWidget {
                   break;
                 case BubbleAction.help:
                   _launchURL("https://discord.gg/9Q2xaN37tV");
+                  break;
+                case BubbleAction.debug:
+                  final deviceInfoPlugin = DeviceInfoPlugin();
+
+                  final dynamic deviceInfo;
+                  if(Platform.isLinux) {
```

Comment:
I'm not a fan of this switch, this won't work once we add mobile devices...

From what I saw in the documentation for 3.2.0 you can just do :
```dart
final deviceInfoPlugin = DeviceInfoPlugin();
final deviceInfo = await deviceInfoPlugin.deviceInfo;
final map = deviceInfo.toMap();
// Push [map] to your service.
```
and this would return all the information that it has on the device, no matter the platform